frontend/src/components/common/link/link.tsx

```diff
@@ -5,7 +5,7 @@ import { NavLink } from 'react-router-dom';
 import styles from './styles.module.scss';
 
 type Props = {
-  to: AppRoute;
+  to: AppRoute | string;
```

Comment:
```suggestion
  to: AppRoute;
```
cust a type where you need it `as AppRpute`